fhir2dataset/api_caller.py

```diff
@@ -99,7 +102,7 @@ def _get_count(self, url):
         response = requests.get(url_number, auth=self.auth)
         logger.info(f"Get {url_number}")
         try:
-            count = min(response.json()["total"], 10000)
+            count = response.json()["total"]
```

Comment:
same here, you can use `count = response.json().get("total")` to avoid the try/except

Reply:
Yes thanks, what do you think of the code now ?